concepts/05 UI Components/PivotGrid/00 Getting Started with PivotGrid/05 Bind the PivotGrid to Data.md

```diff
@@ -0,0 +1,158 @@
+The PivotGrid component can work with data from local arrays, OLAP services (Microsoft SQL Server Analysis Services), or any other data sources. Regardless of the data source type, use the [dataSource](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#dataSource) property to bind the PivotGrid to data. This property accepts a [PivotGridDataSource](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/) instance or configuration object.
+
+PivotGridDataSource allows you to process data before displaying it in the PivotGrid. At its core, PivotGridDataSource contains a [store](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/store/) that encapsulates data access logic. Different store types should be used with different data source types.
```

Comment:
> PivotGridDataSource allows you to process data before displaying it in the PivotGrid.

Can we rewrite this as follows:

> PivotGridDataSource allows you to process data before the PivotGrid displays it.

?

> **At its core**, PivotGridDataSource contains a [store](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/store/) that encapsulates data access logic.

This is a strange starting clause. Even if it may be good in terms of logic, it looks like a part of a narrative-based approach we strive to avoid. Moreover, it may be not very clear to what "its" refers. I would have rewritten this sentence.